lib/project.js

```diff
@@ -361,6 +361,14 @@ BT.projectManager = SC.Object.create({
       this.serverIndex(req, res);
       hasServed = true;
     }
+    if (path === "/socket_broadcaster") {
```

Comment:
I would prefer to have something like debug_broadcaster, or something that is a bit more indicative on how it is supposed to be used.